src/content/docs/concepts/security/csp.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: Content Security Policy (CSP)
+---
+
+Tauri restricts the [Content Security Policy] (CSP) of your HTML pages.
+This can be used to reduce or prevent impact of common web based vulnerabilities
+like cross-site-scripting (XSS). 
+
+Local scripts are hashed, styles and external scripts are referenced using a cryptographic nonce,
+which prevents unallowed content from being loaded.
+
+:::caution
+Avoid loading remote content such as scripts served over a CDN as they introduce an attack vector.
+In general any untrusted file can introduce new and subtle attack vectors.
+:::
+
+The CSP protection is only enabled if set on the Tauri configuration file.
+You should make it as restricted as possible, only allowing the webview to load assets
+from hosts you trust, and preferably own.
+At compile time, Tauri appends its nonces and hashes to the relevant CSP attributes automatically
+to bundled code and assets, so you only need to worry about what is unique to your application.
+
```

Comment:
Add an example of the config file with CSP activated

Reply:
I would like to take this to a later PR/issue, as we should expand more on the `devCSP`, `CSP` and provide more examples of threats, bad and good csp etc.